### PR TITLE
Unit Test for Magento\Fedex\Plugin\Block\DataProviders\Tracking\ChangeTitle

### PR DESCRIPTION
**Pull request by karyna-t**
### Description (*)
Covers Magento\Fedex\Plugin\Block\DataProviders\Tracking\ChangeTitle class with a unit test.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Fedex/Test/Unit/Plugin/Block/DataProviders/Tracking/ChangeTitleTest.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
```
Comment (edenduong) [inline app/code/Magento/Fedex/Test/Unit/Plugin/Block/DataProviders/Tracking/ChangeTitleTest.php:6]: Could you please added a `declare(strict_types=1);` to all new PHP files ?

**Review by edenduong — CHANGES_REQUESTED** [review]

File: app/code/Magento/Fedex/Test/Unit/Plugin/Block/DataProviders/Tracking/ChangeTitleTest.php (hunk 1 above)
Comment (karyna-t) [reply app/code/Magento/Fedex/Test/Unit/Plugin/Block/DataProviders/Tracking/ChangeTitleTest.php:6]: added, thanks

**Review by edenduong — APPROVED** [review]
Thank you for your contribution !

**Comment by magento-engcom-team** [comment]
Hi @edenduong, thank you for the review.
[ENGCOM-6680](https://jira.corp.magento.com/browse/ENGCOM-6680) has been created to process this Pull Request

**Review by lbajsarowicz — APPROVED** [review]
Good job! : )

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6680](https://jira.corp.magento.com/browse/ENGCOM-6680) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**